images/gcp/centos-7.yaml

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
would you mind changing the file to centos79.yaml? 

 
I'm trying to be specific in the point versions we support and I think it's a good practice to get into.

Reply:
This breaks existing files. I would prefer if we do this in another PR and just settle this holistically and not for just this one case.

Reply:
Ha i havent pulled those changes. Alright then no problem.